src/components/pages/deployments/flyouts/deploymentDelete/deploymentDelete.scss

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/variables';
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.delete-deployment-container {
+  @include rem-fallback(font-size, 14px);
```

Comment:
Does this really need to be a different font size than the rest of the content? Even it is, modals should have a default font size... set via the modal's css (and not needed to be set in each individual one).

Reply:
done